### PR TITLE
Wind stats addition

### PR DESCRIPTION
**Pull request by o-gent**
Pretty simple addition to add wind stats to the vehicle class. Tried to follow convention seen in other attribute classes and methods

Needed it for a project I'm doing so maybe it could be useful for other people!

Tested with an STL and is functional

### CODE REVIEW

**Comment by o-gent** [comment]
Those tests look pretty messed up, probably worth just removing any failing tests - looks like they've been failing for a long time. test_goto.py is fairly poor quality anyway but fails when trying to change mode for some reason.. Create tasks for code to be tested. I tried to get nosetests set up but wouldn't run anything even though I'm using the same command as travis-ci

**Comment by o-gent** [comment]
The tests that are failing (from wayyy before my commit) are actually working on my SITL. Looks like dronekit-sitl isn't working. Could make the tests use the manually installed SITL then don't need to fix dronekit-sitl?

**Comment by o-gent** [comment]
Hi @peterbarker, is there anything else I can do to get this integrated?

File: dronekit/__init__.py [hunk 1]
```diff
@@ -1678,6 +1710,13 @@ def listener(self, attr_name, value):
         """
         return self._location
 
+    @property
+    def wind(self):
+        """
+        Current wind status (:pu:class: `Wind`)
+        """
+        return Wind(self._wind_direction, self._wind_speed, self._wind_speed_z)
```
Comment (peterbarker) [inline dronekit/__init__.py:1718]: Shouldn't this, like battery, return `None` if the attributes are all `None`?

Comment (o-gent) [reply dronekit/__init__.py:1720]: Done

**Comment by peterbarker** [comment]
@o-gent new changes have been tested?

**Comment by o-gent** [comment]
@peterbarker I've tested it on my STL
image
but I haven't written a test as it seems the STL used in testing isn't working the same as what I see in my STL

**Comment by peterbarker** [comment]
Thanks @o-gent , merging.

**Pull request merged**